src/components/AllCSS/addBook.css

```diff
@@ -0,0 +1,195 @@
+.book-form {
+  width: 75%;
+  height: 14%;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.book-info {
+  display: flex;
+  flex-direction: column;
+  height: 140px;
+  margin: unset;
+}
+
+.action {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 1.175rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+  padding: 5px;
+}

```

Comment:
- [ ] In my opinion, you don't need to add these properties, because the values of these properties are `normal` by default 👍 So I suggest removing them To keep your code clean and Follow the best practice 💯
```css
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
  letter-spacing: normal;
  ```

> 💡 For more info, [Check this Article](https://speckyboy.com/good-bad-css-practices/)